docs/debugging.md

```diff
@@ -125,13 +125,9 @@ If you want to test dotnet.exe explicitly, so you don't have to worry about whet
 
 ### Debugging NuGet Command Xplat Functionality (add-package/remove-package/list package)
 
-Functionality such as `dotnet.exe add package` or `dotnet list package`, is implemented in [NuGet.CommandLine.XPlat](../src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj).
```

Comment:
I think this sentence is still valuable to keep, especially if we take my suggested heading change.  I'd go even one further and change it to:

```suggestion
All of the `dotnet nuget` commands, in addition to some others, such as `dotnet add package`, dotnet list package`, and `dotnet search`, are implemented in [NuGet.CommandLine.XPlat](../src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj).
```